runner/src/main/kotlin/runner/Runner.kt

```diff
@@ -0,0 +1,64 @@
+package runner
+
+import interpreter.Environment
+import interpreter.InterpretOutput
+import interpreter.Interpreter
+import interpreter.readInputFunction.ReadInputFunction
+import logger.ErrorLogger
+import logger.Logger
+import parser.Parser
+import parser.result.FailureResult
+import parser.result.ParserResult
+import parser.result.SuccessResult
+import reader.StatementFileReader
+import token.Token
+
+class Runner {
+    fun run(
+        reader: StatementFileReader,
+        parser: Parser,
+        interpreter: Interpreter,
+        handler: ErrorLogger,
```

Comment:
I think we should use the Logger interface for both the error handler and the "printing" logger, because if we use ErrorLogger we cannot pass a custom implementation for the handler (ie: A collector)

Reply:
I made it that way so that you can call getError later

Reply:
Yeah but what if I want to pass a handler that logs the error every time? Or adapt the handler in the tck to be able to pass it to this runner?